IFC4x3/Properties/c/CantRotationAxis_1IburP5x9EXf88H5s5Xwyq/DocProperty.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CantRotationAxis_1IburP5x9EXf88H5s5Xwyq" Name="CantRotationAxis" UniqueId="52978d59-17b2-4e86-9208-445d8587af34">
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CantRotationAxis_1IburP5x9EXf88H5s5Xwyq" Name="CantRotationAxis" UniqueId="52978d59-17b2-4e86-9208-445d8587af34" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
 	<Documentation></Documentation>
```

Comment:
That wasnt one of mine, and it seems to be orphaned...

Reply:
its not orphaned, its in a Railway Pset called Pset_RailwayAlignmentCommon attached to IfcAlignment. as an alignment person, i leave to you to decide on its fate if you want me to remove it then let me know

Reply:
@SergejMuhic what would you like me to do with this and Pset_RailwayAlignmentCommon???

Reply:
I think we can remove it for now as none of us knows what it is about. We are in the harmonization process with rail anyway. I think it will be resolved sooner or later.